### ISSUE TITLE
[Anon] Bug report: Chaos - Nurgle.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/warhammer-age-of-sigmar#837]
**File:** Chaos - Nurgle.catz

**BattleScribe version:** 2.01.17

**Platform:** Android

**Dropbox:** No

**Description:** Blight Cyst Battalion needs to be fixed

gives you errors when trying to choose two artefacts.
Warscrolls view smashes your army into one page and completely looks confusing.
forces you to add extra leaders and battleline outside of what you inputted in your cyst making you have extra unnecessary units and leaders causing too many points.

**Comment by UrsinePatriarch** [comment]
Double-checked the above;
1) Correct, the artefacts thing will be fixed; it's not repeating additional artefacts properly for some reason, as it should.
2) This isn't actually something we can fix at this time due to how Battlescribe is formatted.
3) This is just missing keywords that would function as Leaders and Battleline.

Tl;dr these are issues that are easily fixed, so please be patient and we'll attempt to rectify them.
